README.md

```diff
@@ -26,6 +26,7 @@ Two ways to do this are using BRAT or manually.
   $ npm i
   $ npm run build
   ```
+2. Restart Obsidian.
 2. Go to Settings in Obsidian and enable `Helix Keybindings`.
```

Comment:
I know this is a long shot, since this is a very old PR, but I've just taken over the project.

```suggestion
2. Restart Obsidian.
3. Go to Settings in Obsidian and enable `Helix Keybindings`.
```

Would you mind changing the next step digit so that people who read the markdown code directly can follow it properly?



Thanks!!